etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+CONSUL_ROOT="http://localhost:8500/v1/kv/nginx"
+CHALLENGE_PATH="/.well-known/acme-challenge"
+
+function deploy_challenge {
+    local DOMAIN="${1}" TOKEN_FILENAME="${2}" TOKEN_VALUE="${3}"
+
+    curl -sX PUT -d "${TOKEN_FILENAME}" ${CONSUL_ROOT}/domains/${DOMAIN}/acme/challenge/token-filename > /dev/null
```

Comment:
I'd suggest avoiding piping these to `/dev/null` particularly during the development process. Consul isn't very chatty as it is and it'd be good to know that something went wrong. Maybe we pipe them to a `log` function somewhere in this same script?